website/snippets/quickstarts/test-and-document-your-project.md

```diff
@@ -0,0 +1,15 @@
+## Add tests to your models
+
+<Snippet src="tutorial-add-tests-to-models" />
+
+## Document your models
+
+<Snippet src="tutorial-document-your-models" />
+
+3. Click the book icon in the Develop interface to launch documentation in a new tab.
```

Comment:
Does the snippet mention that you need to run `dbt docs generate` before the book icon will activate?